libs/website/ui/introduction/welcome/welcome.section.tsx

```
@@ -0,0 +1,47 @@
+import type { ThreeElements } from '@react-three/fiber'
+import type * as THREE from 'three'
+import { Canvas, useFrame } from '@react-three/fiber'
+import React, { useRef, useState } from 'react'
+import { WELCOME_CONSTATNTS } from '../constants/welcome.constants'
+import { Welcome } from './welcome'
+
+function Box(props: ThreeElements['mesh']) {
+  const meshRef = useRef<THREE.Mesh>(null!)
+  const [hovered, setHover] = useState(false)
+  const [active, setActive] = useState(false)
+  useFrame((state, delta) => (meshRef.current.rotation.x += delta))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)